server/jetstream_cluster.go

```diff
@@ -7640,12 +7641,6 @@ func (s *Server) jsClusteredConsumerRequest(ci *ClientInfo, acc *Account, subjec
 
 		// We need to set the ephemeral here before replicating.
 		if !isDurableConsumer(cfg) {
-			// We chose to have ephemerals be R=1 unless stream is interest or workqueue.
-			// Consumer can override.
-			if sa.Config.Retention == LimitsPolicy && cfg.Replicas <= 1 {
```

Comment:
Where is this logic now handled?

Reply:
That's already handled by `cc.createGroupForConsumer`
```go
	// First shuffle the active peers and then select to account for replica = 1.
	rand.Shuffle(len(active), func(i, j int) { active[i], active[j] = active[j], active[i] })
	peers = active[:replicas]
}
...
return &raftGroup{Name: groupNameForConsumer(peers, storage), Storage: storage, Peers: peers}
```